admin/src/css/theme.less

```diff
@@ -145,3 +145,9 @@ ul {
 .btn-bottom-spacing  {
   margin-bottom: 5px;
 }
+.partner-image {
+  img {
+    height: 150px;
+    width: 150px;
```

Comment:
This is duplicated in the webapp css below - can you instead include it somewhere where both apps will get it so we can reuse it?